test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -660,9 +660,11 @@ def _cleanup(self):
             ftd.extend([f for f in files if bn(f).startswith('pbsscrpt')])
             ftd.extend([f for f in files if bn(f).startswith('pbs.conf.')])
             ftd.extend([f for f in files if p.match(bn(f))])
+            ftd.extend(du.tmpfilelist)
```

Comment:
Actually, I think this needs to be removed from the for loop.
As of right now, it will repeat this for every tmpdir in dirlist (line 651). If there's multiple, then the files in `du.stmpfilelist` will be added to `ftd` multiple times and PTL will try to delete it multiple times.

Reply:
Ah. yes. you are right. I moved it outside 'for' loop